public/js/pimcore/helpers.js

```diff
@@ -1035,34 +1035,100 @@ pimcore.helpers.uploadDialog = function (url, filename, success, failure, descri
         emptyText: t("select_a_file"),
         fieldLabel: t("file"),
         width: 470,
-        name: filename,
+        name: filename+'[]',
         buttonText: "",
         buttonConfig: {
             iconCls: 'pimcore_icon_upload'
         },
         listeners: {
             change: function (fileUploadField) {
-                if(fileUploadField.fileInputEl.dom.files[0].size > pimcore.settings["upload_max_filesize"]) {
-                    pimcore.helpers.showNotification(t("error"), t("file_is_bigger_that_upload_limit") + " " + fileUploadField.fileInputEl.dom.files[0].name, "error");
-                    return;
-                }
+                var win = new Ext.Window({
```

Comment:
All the `var` should be `let/const`
The rest LGTM

Reply:
@kingjia90 Done. Sorry, was several weeks on sick-leave.